herbert_core/utilities/classes/@unique_references_container/unique_references_container.m

```diff
@@ -280,6 +280,9 @@
             for ii = 1:val.n_objects
                 obj = val.get(ii);
                 [obj,hash] = build_hash(obj);
+                % as this code updates self and hence glc, need to refetch it every
+                % time here
+                glc = self.global_container('value',self.global_name);                
```

Comment:
This is not a fixture. 

The previous implementation was adding 23 runs to the container. 
And this one clear it up at every header calulation. 
Experiment for this run comes with 1 run where should be 23 runs -- one unique and 22 duplicates. 

I am surprized that side-effects are not reflected in tests. Probably Tobyfit does not yet uses all these instruments properly and everything works on IX_experiment which is not a conteiner